api/geth_backend.go

```diff
@@ -912,7 +912,8 @@ func (b *GethStatusBackend) AppStateChange(state string) {
 			return
 		}
 
-		if !localNotifications.IsWatchingWallet() {
+		// If we have no local notifications, force wallet stop, otherwise check if it's watching the wallet
+		if localNotifications != nil || (localNotifications != nil && !localNotifications.IsWatchingWallet()) {
```

Comment:
first on should be localNotifications **==** nil  ?

Reply:
dang